docs/guide/personalization.md

```diff
@@ -119,42 +105,48 @@ ez_recommendation.export.users_authentication.login: '<customerID>'
 ez_recommendation.export.users_authentication.password: '<licenseKey>'
 ```
 
-Place this in a settings file which won't be affected by an update to the Recommendation bundle via Composer.
+Place this in a settings file which won't be affected by an update to the bundle.
 
 ## Configuration
 
-### 1. Define what content should be tracked and exported
+### Set up Content Type tracking
 
-Visitor events (clicks, buys, ...) on the site need to be sent to the Personalization Solution for the recommendations to be calculated. The content types that are marked to be tracked are also exported to the Personalization Engine. Please note that you can only recommend what you track!
+Visitor events (clicks, buys, etc.) on the site need to be sent to the Recommendation engine for the recommendations to be calculated.
+The Content Types that are tracked are also exported to the engine.
+Tracking Content Types is required for displaying recommendations.
 
-By defining the Content Types in the local `app/config/config.yml` file, the content will be initially exported by a script. After this, it will be kept in sync with the Personalization Solution everytime a change occurs in the eZ Platform back office.
+You define Content Types to track in the local `app/config/config.yml` file.
+The content will then be initially exported by a script.
+After this, it will be kept in sync with the Personalization Solution every time a change occurs in the eZ Platform Back Office.
 
 The bundle's configuration is SiteAccess-aware. This is an example of the settings (in `config.yml`):
 
 ``` yaml
 ez_recommendation:
     system:
-        default:
-            yoochoose:
+        <siteaccess>:
+            recommendation:
                 customer_id: 12345
                 license_key: 1234-5678-9012-3456-7890
-            server_uri: "http://example.com"
+            server_uri: http://example.com
             recommender:
                 included_content_types: [blog, article]
```

Comment:
This one should be under the recommendation key (the same as customer_id and license_key)